.codecov.yml

```diff
@@ -1,10 +1,14 @@
+codecov:
+  require_ci_to_pass: false
```

Comment:
I hope this will make codecov act on its own, not coupling it to other builds statuses